tests/wpt/mozilla/tests/css/table_rowspan_rowgroup_a.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <meta charset="UTF-8">
+  <link rel="match" href="table_rowspan_rowgroup_ref.html">
```

Comment:
Could a `<link rel="spec" href="https://html.spec.whatwg.org/multipage/tables.html#attributes-common-to-td-and-th-elements">` be a good idea here?